src/Rector/Contrib/PHPUnit/SpecificMethod/AssertTrueFalseToSpecificMethodRector.php

```diff
@@ -14,10 +14,12 @@
 
 /**
  * Before:
- * - $this->assertTrue(is_readable($readmeFile), 'messag');
+ * - $this->assertTrue({function}($something), 'messag');
+ * - $this->assertFalse({function}($something), 'messag');
  *
  * After:
- * - $this->assertIsReadable($readmeFile, 'message'));
+ * - $this->assert{$function}($something, 'message'));
+ * - $this->assertNot{$function}($something, 'message'));
```

Comment:
These comments should be before/after examples. As less generic as possible, so people understand it with away.

`assert{$function}` could be anything